tests/SparseFieldsetsTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Spatie\Fractalistic\Test;
+
+use Spatie\Fractalistic\Fractal;
+use Spatie\Fractalistic\ArraySerializer;
+
+class SparseFieldsetsTest extends TestCase
+{
+    /** @test */
+    public function it_can_filter_out_fields()
+    {
+        $array = Fractal::create()
+            ->item([
+            'a' => 'A',
```

Comment:
This example isn't as clear as it could be. Could you replace the keys with `key1`, `key2`, `key3` and the values with `value1`, `value2`, `value3`?